cmd/daemon/kubernetes/pods.go

```diff
@@ -267,5 +267,16 @@ func alertSquad(squad string, annotations map[string]string) (alertChannel strin
 		}
 		return value
 	}
+
+	if destination, ok := squadCustomAlertChannel[squad]; ok {
+		return destination
+	}
+
 	return fmt.Sprintf("#squad-%s-alerts", squad)
 }
+
+var (
+	squadCustomAlertChannel = map[string]string{
+		"atlas": "squad-atlas-critical-alerts",
```

Comment:
This should be a flag in the CLI so that is customisable when we define its deployment

Reply:
I've converted the PR back into draft, so let's adjust it to your requests as they make more sense. Thank you! :)